plugins/pipelines/app/decorators/stage_decorator.rb

```diff
@@ -0,0 +1,9 @@
+Stage.class_eval do
+  serialize :next_stage_ids, Array
+
+  def next_stage
```

Comment:
that button is then useless, right ?

On Tue, Jun 2, 2015 at 12:04 PM, Shane Hender <notifications@github.com>
wrote:

> In plugins/pipelines/app/decorators/stage_decorator.rb
> <https://github.com/zendesk/samson/pull/425#discussion_r31558956>:
>
> > @@ -0,0 +1,9 @@
> > +Stage.class_eval do
> > +  serialize :next_stage_ids, Array
> > +
> > +  def next_stage
>
> this function is used for that button on deploy page, and only expects one
> stage. So didn't want to change it for this PR.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/425/files#r31558956>.
>